libs/sdk-ui-tests/stories/visual-regression/visualizations/insightStories.tsx

```diff
@@ -137,14 +137,14 @@ const DoNotRenderConfigPanel = "this-classname-should-not-exist-in-the-document"
 /*
  * This ready resolver returns true when both are true:
  *
- * -  3 visualizations are rendered
+ * -  4 visualizations are rendered
```

Comment:
seems like somebody has forgotten to increase that when adding one more instance